server/src/com/cloud/configuration/Config.java

```diff
@@ -664,6 +664,14 @@
             "/var/cloudstack/mnt",
             "The mount point on the Management Server for Secondary Storage.",
             null),
+    Saml2FailedLoginRedirectUrl(
+            "Advanced",
```

Comment:
@GabrielBrascher please move this to a ConfigKey, existing deployments may not have this config created.